projects/kubernetes/e2e/Makefile

```diff
@@ -0,0 +1,29 @@
+ORG?=docker
+IMAGE=kubernetes-for-desktop-e2e
+HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
+
+kubernetes_version=v1.8.0
+
+kube_release_artefacts=https://dl.k8s.io/$(kubernetes_version)/bin/linux/amd64
+
+.PHONY: build
+build: _cache/kubernetes-test.tar.gz _cache/kubectl
+	docker build --network=none -t $(ORG)/$(IMAGE):$(HASH) .
+
+.PHONY: run
+run: build
+	docker run -ti -v /var/lib/kubeadm:/etc/kubernetes $(ORG)/$(IMAGE):$(HASH)
+
+_cache/kubectl:
+	mkdir -p _cache
+	curl -fSL -o $@ https://dl.k8s.io/$(kubernetes_version)/bin/linux/amd64/kubectl
```

Comment:
Does d4m at least not already supply a `kubectl`?

Reply:
Yes, but we need it inside the image, as we have namespace cleanup script and `e2e.test` shells out to it too.